Jint/SourceParsedEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Esprima.Ast;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Jint
+{
+    public class SourceParsedEventArgs
```

Comment:
should be sealed if not an extension point, maybe even a record 😉

Reply:
Sealed, absolutely. But philosophy-wise, here I'm quite sure record doesn't add much, though - can't remember ever needing to compare two eventargs. 🙂 (and since some EventArgs may oftentimes need to be mutable, I'll generalize and say no EventArgs should be records, even when they're immutable 😉).